src/test/SimpleLogin.java

```diff
@@ -1,62 +1,47 @@
     public static String username = "morad";
     public static String password = "morad";
 
-    public static void main(String[] args) throws Exception {
-        SimpleLogin SL = new SimpleLogin();
-        SL.HTTPStart(); // -> For HTTP Servers (default port: 80)
-        /* For HTTPS Servers
-        SL.setPort(443);
-        SL.HTTPSStart("./etc/keystore.jks", "123456");
-        */
-    }
+    private static HttpResponse handle(HttpRequest httpRequest) {
+        HttpResponse httpResponse = new HttpResponse();
 
-    @Override
-    public HashMap<String, Object> main(HashMap<String, String> headers, byte[] body) {
-        HashMap<String, Object> response = new HashMap<>();
-        response.put("mime", "text/html"); // By default, the Content-Type is text/html
-
-        if (headers.get("path").equals("/index.html")) {
-            response.put("body", HTMLCode.getBytes());
-            response.put("code", HTTPCode.OK);
-        } else if (headers.get("path").equals("/api/login")) {
-            // Does not matter if the request is POST or GET
-            HashMap<String, String> LoginInfo = JSON.QHM(new String(body));
+        if (httpRequest.getPath().equals("/index.html")) {
+            httpResponse.setHttpContentType(HttpContentType.TEXT_HTML);
+            httpResponse.setBody(HTMLCode);
+            httpResponse.setHttpStatusCode(HttpStatusCode.OK);
+        } else if (httpRequest.getPath().equals("/api/login") && httpRequest.getHttpRequestMethod() == HttpRequestMethod.POST) {
+            httpResponse.setHttpContentType(HttpContentType.APPLICATION_JSON);
+            HashMap<String, String> LoginInfo = JSON.QHM(httpRequest.getBody());
             if (!(LoginInfo.containsKey("username") && LoginInfo.containsKey("password"))) {
-                response.put("body", "Invalid Request.".getBytes());
-                response.put("code", HTTPCode.BAD_REQUEST);
+                httpResponse.setBody("{\"success\": \"false\"}");
             } else {
                 if (LoginInfo.get("username").equals(username) && LoginInfo.get("password").equals(password)) {
-                    response.put("body", JSON.HMQ(new HashMap<>() {{
-                        put("status", "success");
-                    }}).getBytes());
+                    httpResponse.setBody("{\"success\": \"true\"}");
```

Comment:
Why don't you use JSON class?

Reply:
Please add an example for a custom header that changes every request, to clarify the impl.

Reply:
> Why don't you use JSON class?

Just an example.

Reply:
> Please add an example for a custom header that changes every request, to clarify the impl.

```java
httpResponse.addHeader("key", "value");
```

Reply:
I don't know how to use it and to be honest I don't like it.